### PR TITLE
Refactor directory structure: moving shared lib files to pkg

### PR DESCRIPTION
**Pull request by yokawasa**
### Description

Move the following shared libs to `pkg` from `controllers`

- cloudstorages/*.go
- notificationservices/*.go
- utils/*.go
- command.go + command_test.go

No additional code and no logic changed

This is  a preparatory PR for supporting new providers (such as [storages](https://github.com/st-tech/gatling-operator/issues/38) or notifications )

### Test

- make test: OK
- CI / Test : [OK](https://github.com/st-tech/gatling-operator/runs/7492451531?check_suite_focus=true)
- manual integration test (not automated yet): OK

### CODE REVIEW

**Review by takuya-mikami — APPROVED** [review]
LGTM

**Review by taquaki-satwo — APPROVED** [review]
LGTM!

File: pkg/commands/commands.go [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 import (
 	"fmt"
 
-	cloudstorages "github.com/st-tech/gatling-operator/controllers/cloudstorages"
+	cloudstorages "github.com/st-tech/gatling-operator/pkg/cloudstorages"
 )
 
-func getGatlingWaiterCommand(parallelism *int32, gatlingNamespace string, gatlingName string) string {
+func GetGatlingWaiterCommand(parallelism *int32, gatlingNamespace string, gatlingName string) string {
```
Comment (akitok) [inline pkg/commands/commands.go:9]: Why did you change from lowerCamelCase to UpperCamelCase?

Comment (yokawasa) [reply pkg/commands/commands.go:9]: Starting from Capital means allowing public access, which is needed as the command functions are move on to pks and set different package name like `commands` to make it shared library

Comment (yokawasa) [reply pkg/commands/commands.go:9]: In other other, command functions are no longer accessible in local scope

Comment (akitok) [reply pkg/commands/commands.go:9]: I got it! thanks.

**Review by akitok — APPROVED** [review]
Looks good to me!

**Comment by yokawasa** [comment]
@taquaki-satwo @takuya-mikami @akitok 
thanks for the review !
